src/stories/AxisSystem.stories.tsx

```diff
@@ -44,18 +44,49 @@ export const AspectRatio = Template.bind({});
 
 AspectRatio.args = {
   abscissaConfig: { indexDomain: [0, 10], showGrid: true },
-  ordinateConfig: { indexDomain: [0, 1], showGrid: true },
-  aspectRatio: 10,
+  ordinateConfig: { indexDomain: [0, 2], showGrid: true },
+  aspectRatio: 10 / 2,
 };
 
 export const NoGrid = Template.bind({});
 
 NoGrid.args = {
-  abscissaConfig: { indexDomain: [-5, 20] },
-  ordinateConfig: { dataDomain: [0, 1] },
+  abscissaConfig: { indexDomain: [-5, 20], showGrid: false },
+  ordinateConfig: { dataDomain: [0, 2], showGrid: false },
```

Comment:
Shame we can't use [Controls](https://storybook.js.org/docs/react/essentials/controls) with nested args...

Reply:
You can define custom controls, but there's an ongoing issue on the add-on for supporting nested objects automatically, so I didn't bother.

For now, my goal is more to make sure components work in isolation. We can improve the documentation aspect later on.

Reply:
Let me rephrase that: "Shame we can't use Controls easily because of the nested args" ^^